frontend/catalyst/api_extensions/__init__.py

```diff
@@ -35,7 +35,7 @@
 from catalyst.api_extensions.error_mitigation import mitigate_with_zne
 from catalyst.api_extensions.function_maps import vmap
 from catalyst.api_extensions.quantum_operators import (
-    Adjoint,
+    HybridAdjoint,
```

Comment:
Just a reminder to add this PR to the changelog before merging ;)

Reply:
Good point!

Reply:
Done 👍